draft-west-cookie-incrementalism.md

```diff
@@ -199,6 +231,70 @@ protecting developers by making cross-site delivery an explicit choice, as oppos
 default.
 
 
+### "Lax-Allowing-Unsafe" Enforcement {#lax-allowing-unsafe}
+
+The "Lax" enforcement mode described in Section 5.3.7.1 of {{RFC6265bis}} allows a cookie to be sent
+along with cross-site requests if and only if they are top-level navigations with a "safe" HTTP
+method. In certain cases it may be desirable for a cookie to be excluded from non-top-level
```

Comment:
I think it's reasonable to strengthen this a bit. Perhaps something like "Implementation experience shows that this is difficult to apply across the board, and it may be reasonable to temporarily carve out cases in which cookies that rely on today's `Default` behavior can continue to be delivered as we shift the default to `Lax`."

Reply:
Cool, sounds good. I'll use that.